src/decomon/backward_layers/backward_layers.py

```diff
@@ -355,6 +363,43 @@ def get_config(self):
         )
         return config
 
+    def get_affine_components(self, x):
+        """Conv is a linear operator but its affine component is implicit
```

Comment:
Great description. But if we are to follow [PEP-257](https://peps.python.org/pep-0257/#multi-line-docstrings), "Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description."

So maybe something like
```python
"""Express the implicit affine matrix of the convolution layer.

Conv is a linear operator but its affine component is implicit
we use im2col and extract_patches to express the affine matrix
Note that this matrix is Toeplitz

Args:
     x: list of input tensors
Returns:
     the affine operators W, b : conv(x)= Wx + b
"""
```

would be better?